src/components/notification/checker.ts

```diff
@@ -188,6 +204,11 @@ export const notificationChecker = async (notifications: Notification[]) => {
     .map((notif) => notif.data as WebhookData)
     .map(webhookNotificationInitialChecker)
 
+  const discordWebhookNotification = notifications
+    .filter((notif) => notif.type === 'discordWebhook')
```

Comment:
Hi @galanggg thank you for your contribution!

Is there any other way to connect into Discord besides Webhook? If there is not, I think it's better renamed as `discord` rather than `discordWebhook` to make it uniform like other types (e.g `slack`, `teams`, `discord`)

Reply:
Hi hi @dennypradipta, thank you for the review!

For now, Discord only supports webhook. Yup, I also think it's better to change to `discord` to make it look clearer. Thank you for the feedback also :pray: